src/fu-tool.c

```diff
@@ -548,10 +549,40 @@ fu_util_install_task_sort_cb (gconstpointer a, gconstpointer b)
 	return fu_install_task_compare (task1, task2);
 }
 
+static gboolean
+fu_util_download_out_of_process (const gchar *uri, const gchar *fn, GError **error)
+{
+	const gchar *argv[] = { "curl", uri, "--output", fn, NULL };
```

Comment:
Is curl already part of flatpak environment or does fwupd flatpak need to package it internally?
I guess same question for if it's part or core snap, I'll check.

Reply:
It's in the GNOME runtime already.

Reply:
OK.  I think something will need to be added to the snap to install this.
At least in Ubuntu-land, `/usr/bin/curl` isn't installed by default.  `/usr/bin/wget` is though.  What do you think of trying `curl` first and falling back to `wget` if it doesn't exist?

I think the behavior is pretty similar.  If you switch to use `-o` instead of the full word the command line options I think are identical too.

Reply:
I think it would be pretty counteritutive to have different behavior if you are running a flatpaked fwupdtool vs /usr/libexec/fwupd/fwupdtool vs snapped fwupdtool.